src/commonTest/kotlin/fr/acinq/lightning/payment/PaymentPacketTestsCommon.kt

```diff
@@ -657,5 +657,12 @@ class PaymentPacketTestsCommon : LightningTestSuite() {
+        val (_, _, trampolineOnion) = OutgoingPaymentPacket.buildTrampolineToNonTrampolinePacket(invoice, NodeHop(randomKey().publicKey(), randomKey().publicKey(), CltvExpiryDelta(444), 0.msat), finalAmount, finalExpiry)
+        assertTrue(trampolineOnion.packet.payload.size() < 1142)
```

Comment:
We should rather build the final payment onion that contains the trampoline onion to prove that it has the correct size?

Reply:
Done